Expensify/App
# Fix extra Text Input padding in report action compose (Android) #1002

### Details
Centred the submit button on the chat box.   
### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes #1002 

### Tests

1. Open the Expensify.cash app
2. Log in
3. Tap the hamburger menu button( ☰ ) in the top-left
4. Start a new DM (or select an existing DM)
5. You will notice that the submit button has some reasoned padding from the bottom. 

### Tested On

- [x ] Android

### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>


#### Web


#### Mobile Web

#### Desktop


#### iOS


#### Android

![Screenshot 2020-12-25 at 8 54 16 PM](https://user-images.githubusercontent.com/4520893/103140772-7c386500-46f3-11eb-8af2-79f4e94389a7.png)
